frenai/chain.json

```diff
@@ -0,0 +1,96 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "frenai",
+  "status": "live",
+  "website": "https://fren.ai",
+  "network_type": "mainnet",
+  "pretty_name": "fren.ai",
+  "chain_id": "fren-1",
+  "bech32_prefix": "fren",
+  "daemon_name": "frend",
+  "node_home": "$HOME/.frenai",
+  "key_algos": ["secp256k1"],
+  "slip44": 118,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "ufren",
+        "fixed_min_gas_price": 1e-9,
+        "low_gas_price": 1e-9,
+        "average_gas_price": 0.00005,
+        "high_gas_price": 0.0001
+      }
+    ]
+  },
+  "staking": {
+    "staking_tokens": [
+      {
+        "denom": "ufren"
+      }
+    ]
+  },
+  "codebase": {
+    "git_repo": "https://github.com/frenailabs/frenai/",
+    "recommended_version": "v1.0.0",
+    "compatible_versions": ["v1.0.0"],
```

Comment:
Please duplicate this version data into a version history structure (either a `versions` array in this file or `versions.json` file.